src/v2/BaseLoginRouter.js

```diff
@@ -117,8 +100,34 @@ export default Router.extend({
     this.appState.setIonResponse(ionResponse);
   },
 
-  render: function (Controller, options = {}) {
+  handleRemediationError (error = {}) {
+    if (error && error.details && error.details.stateHandle) {
+      // 1. loosely check whether is IDX error response
+      // see idx for details: https://github.com/okta/okta-idx-js/blob/master/src/index.js
+
+      // Need to mimic IdxRespones as idx returns raw response at error case
```

Comment:
do we get a raw response (non idx) only when there is an error for introspect? Should this be offloaded to idx-js layer?

Reply:
at the very least we could rename it to `handleIonResponse `| `parseIonResponse` instead of `handleRemediationSuccess `

Reply:
- yes: https://github.com/okta/okta-idx-js/pull/27#issuecomment-656817087
- will rename method.